File: src/main/java/com/p2p/lending/dao/BidDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Borrowcord;
import com.p2p.lending.entity.Borrowmoney;
import com.p2p.lending.entity.Certification;
import com.p2p.lending.entity.InvestInfo;
import com.p2p.lending.entity.Product;
import com.p2p.lending.entity.Trade;
import com.p2p.lending.entity.Users;
import com.p2p.lending.entity.Withdrawal;

public interface BidDao {

	public List<Product> todaoqi();

	public void upzt(Integer id);

	public List<InvestInfo> totouzilist(Integer id);
	
	public Users seluesr(Integer uid);
	

	// ------------项目到期----------------//
	
	public Certification togetyue(Integer uID);

	public void toupyue(Map<String, String> map);

	public void toaddtrade(Trade trade);
	
	// ------------筹款到期----------------//
	
	public List<Product> togetck();
	
	public void upzts(Integer id);
	
	// ------------首页展示信息----------------//
	
	public List<Borrowmoney> tosize();
	public List<Withdrawal> tosizew();
	public List<Borrowcord> tosizeb();

}

File: src/main/java/com/p2p/lending/dao/BidDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Borrowcord
import com.p2p.lending.entity.Borrowmoney
import com.p2p.lending.entity.Certification
import com.p2p.lending.entity.InvestInfo
import com.p2p.lending.entity.Product
import com.p2p.lending.entity.Trade
import com.p2p.lending.entity.Users
import com.p2p.lending.entity.Withdrawal

interface BidDao {

    fun todaoqi(): List<Product>

    fun upzt(id: Int?)

    fun totouzilist(id: Int?): List<InvestInfo>

    fun seluesr(uid: Int?): Users


    // ------------项目到期----------------//

    fun togetyue(uID: Int?): Certification

    fun toupyue(map: Map<String, String>)

    fun toaddtrade(trade: Trade)

    // ------------筹款到期----------------//

    fun togetck(): List<Product>

    fun upzts(id: Int?)

    // ------------首页展示信息----------------//

    fun tosize(): List<Borrowmoney>
    fun tosizew(): List<Withdrawal>
    fun tosizeb(): List<Borrowcord>

}